src/homebase/datalog_console.cljs

```diff
@@ -0,0 +1,18 @@
+(ns homebase.datalog-console
+  (:require [goog.object :as gobj]
+            [cljs.reader]))
+
+(js/document.documentElement.setAttribute "__datalog-inspect-remote-installed__" true)
```

Comment:
":datalog-console.remote/installed?"

Reply:
Seems that these special characters are not allowed as an attribute on html document. Changed it to "__datalog-console-remote-installed__"